platform/common.go

```diff
@@ -185,10 +185,11 @@ var AllNativeCodes = []string{QDNET, QDJVM, QDJVMC, QDGO, QDPY, QDPYC, QDJS, QDP
 
 func Image(code string) string {
 	if val, ok := DockerImageMap[code]; ok {
-		if code == QDNETC || code == QDCL {
-			return val + releaseVersion + "-eap"
-		}
-		return val + releaseVersion
+		//if code == QDNETC || code == QDCL {
+		//	return val + releaseVersion + "-eap"
+		//}
+		//return val + releaseVersion
+		return val + releaseVersion + "-eap"
```

Comment:
Why have we started to append eap everywhere? I think that in general we need to have another map with only EAP linters and use it here. WDYT?

Reply:
May be we could hide such logic behind some release process procedure/flag? It's very hard to track such parts of code to revert them later.

Reply:
May be we could hide such logic behind some release process procedure/flag? It's very hard to track such parts of code to revert them later.